tests/water_movement_test.cpp

```diff
@@ -261,8 +261,14 @@ struct swim_scenario {
 static int swimming_steps( avatar &swimmer )
 {
     map &here = get_map();
+    // This shouldn't work.
+    CAPTURE( swimmer.pos() );
+    avatar_action::move( swimmer, here, swimmer.pos() + tripoint_west );
```

Comment:
avatar_action::move() wants an offset, not a coordinate, so you should provide just tripoint_west as the third argument.
No idea why it's ending up at z= -2 though?